### PR TITLE
Add Pennsylvania State University

### PR DESCRIPTION
**Pull request by Sunit-Kulkarni**

### CODE REVIEW

**Review by Carlgo11 — COMMENTED** [review]
From looking at http://www.identity.psu.edu/set-up-instructions/#landline it seems like they support phone calls too. Is that right? If so, add `phone: Yes`.

**Comment by Sunit-Kulkarni** [comment]
Good catch. Duo lets you use phone calls.

**Review by Carlgo11 — DISMISSED** [review]

File: _data/education.yml [hunk 1]
```diff
@@ -223,6 +223,16 @@ websites:
...
 
+    - name: Pennsylvania State University
+      url: http://www.psu.edu/
+      img: psu.png
+      tfa: Yes
+      sms: Yes
+      phone: Yes
+      software: Yes
+      hardware: Yes
+      doc: http://www.identity.psu.edu/services/authentication-services/two-factor/self-service-portal/
```
Comment (jamcat22) [inline _data/education.yml]: I think we should change the URL to either their overall service topic on 2FA (http://www.identity.psu.edu/services/authentication-services/two-factor/) which links to several articles, including the one about enabling it, or their 2FA enrollment site (http://get2fa.psu.edu/).

**Review by jamcat22 — DISMISSED** [review]

File: _data/education.yml (hunk 1 above)
Comment (Sunit-Kulkarni) [reply _data/education.yml]: I say use the First Link since it is more general.

Comment (conorgil) [reply _data/education.yml]: @Sunit-Kulkarni I'm hopping in this thread late, but based on the comments from review it sounds like once you update that URL this PR should be good to go

Comment (Sunit-Kulkarni) [reply _data/education.yml]: Alrighty. Just updated it!

**Review by stephengroat — APPROVED** [review]

**Review by mxxcon — APPROVED** [review]

**Pull request merged**